src/screen.cpp

```diff
@@ -228,9 +226,8 @@ void BasicInfoSummary(bool bRedraw)
         #endif
     }
 
-
 #ifndef ARDUINO_HELTEC_WIFI_KIT_32
-    display.drawRect(0, 0, display.width(), display.height(), borderColor);
+    //display.drawRect(0, 0, display.width(), display.height(), borderColor);
 #endif
```

Comment:
If we're not drawing the border any more, then why don't we just cut the whole #ifndef block?